src/client/auth/auth.proto

```diff
@@ -28,29 +28,10 @@ import "google/protobuf/timestamp.proto";
 
 //// Activation API
 
-// ActivateRequest mirrors AuthenticateRequest. The caller is authenticated via
-// GitHub OAuth, and then promoted to the cluster's first Admin. Afterwards, the
-// caller can promote other users to Admin and remove themselves
+// ActivateRequest enables authentication on the cluster. It issues an auth token
+// with no expiration for the irrevocable admin user `pach:root`.
 message ActivateRequest {
-  // If set, Pachyderm will authenticate the caller as this user.
-  // - If set to a github user (i.e. it has a 'github:' prefix or no prefix)
-  //   then Pachyderm will confirm that it matches the user associated with
-  //   'github_token'
-  // - If set to a robot user (i.e. it has a 'robot:' prefix), then Pachyderm
-  //   will generate a new token for the robot user; this token will be the only
-  //   way to administer this cluster until more admins are added.
-  string subject = 2;
-
-  // This is the token returned by GitHub and used to authenticate the caller.
-  // When Pachyderm is deployed locally, setting this value to a given string
-  // will automatically authenticate the caller as a GitHub user whose username
-  // is that string (unless this "looks like" a GitHub access code, in which
-  // case Pachyderm does retrieve the corresponding GitHub username)
-  string github_token = 1 [(gogoproto.customname) = "GitHubToken"];
-
-  // If set, activate auth for the root user (`pach:root`), and add this
-  // token as the initial login token. The root user will have super
-  // admin permissions on the cluster.
+  // If set, this token is used as the root user login token.
```

Comment:
After reviewing the rest of the PR, I now know that it is, so you can ignore this. Or maybe it would make sense to add like "if unset pachd will generate and return a token" or something to the comment?